{"name": "test_single_resource_not_found", "status": "failed", "statusDetails": {"message": "AssertionError: assert None\n +  where None = print({})", "trace": "self = <test_requests.TestGetRequests object at 0x7fe4945d1610>\n\n    def test_single_resource_not_found(self):\n        get_req = requests.get('https://reqres.in/api/unknown/23')\n        get_req_json = get_req.json()\n        assert get_req.status_code == 404\n>       assert print(get_req_json)\nE       assert None\nE        +  where None = print({})\n\ntest_requests.py:45: AssertionError"}, "start": 1612049332168, "stop": 1612049332526, "uuid": "f28d6d2f-5c92-41ec-969d-4db63f44c685", "historyId": "809cba64e21dbb5375ced0b5a8ed5640", "testCaseId": "18d9767c9d6e57814e1c5bd0c27061b3", "fullName": "api_testing.homework3.test_requests.TestGetRequests#test_single_resource_not_found", "labels": [{"name": "parentSuite", "value": "api_testing.homework3"}, {"name": "suite", "value": "test_requests"}, {"name": "subSuite", "value": "TestGetRequests"}, {"name": "host", "value": "kali"}, {"name": "thread", "value": "2929-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "api_testing.homework3.test_requests"}]}